front/src/modules/timesStops/consts.ts

```diff
@@ -0,0 +1,6 @@
+export const marginRegExValidation = /^(\d+(\.\d+)?)(%|min\/100km)$/;
+
+export const MarginUnit = {
+  percent: '%',
+  second: 's',
+} as const;
```

Comment:
Is there a reason not to have `min/100km`? That feels wrong, but I don't really get the full context

Reply:
It wasn’t used, but ok, I added it.